handlers/role_notification_handler.py

```diff
@@ -13,11 +14,12 @@ async def on_message(self, message: discord.Message):
             return
 
         if await role_helper.should_be_notified(message.author):
-            await message.reply(message.author.mention + ' _You don\'t have roles! Roles unlock hidden channels '
+            msg = await message.reply(message.author.mention + ' _You don\'t have roles! Roles unlock hidden channels '
                                                          'related to your university and programme, so you should get '
                                                          'them by going to the <#879407265336152104> channel '
                                                          'and clicking the corresponding buttons._')
-
+            time.sleep(60)
```

Comment:
Instead of `time.sleep`, please consider using asyncio for that, as `time.sleep` would block the event loop.

Reply:
My bad, fixed.